components/xnat2bids/Xnat2bidsfragment.vue

```diff
@@ -73,70 +73,59 @@
 
 <script>
 import { Fragment } from 'vue-fragment'
-import { mapFields, mapMultiRowFields } from 'vuex-map-fields'
+import { mapFields } from 'vuex-map-fields'
 
 export default {
   components: {
     Fragment,
   },
   computed: {
-    ...mapFields([
-      'xnat2bids.version',
-      'xnat2bids.output_path',
-      'xnat2bids.needs_bidsmap',
-      'xnat2bids.bidsmap_dir',
-      'xnat2bids.bidsmap_file',
-      'xnat2bids.overwrite',
-      'xnat2bids.cleanup',
-      'xnat2bids.verbose',
-    ]),
-
-    ...mapMultiRowFields(['xnat2bids.sessions']),
+    ...mapFields(['xnat2bids']),
 
     output_path_validated() {
-      if (this.output_path) {
-        return this.output_path
+      if (this.xnat2bids.output_path) {
+        return this.xnat2bids.output_path
       } else {
         return 'MUST ENTER PATH IN FORM'
       }
     },
 
     sessionDictString() {
-      let key = this.sessions[0].participant_id
+      let key = this.xnat2bids.sessions[0].participant_id
       if (!key) {
         key = 'MUST ENTER IN FORM'
       }
-      let val = this.sessions[0].xnat_id
+      let val = this.xnat2bids.sessions[0].xnat_id
       if (!val) {
         val = 'MUST ENTER IN FORM'
       }
       const lines = [`declare -A sessions=([${key}]="${val}"`]
-      for (let i = 1; i < this.sessions.length; i++) {
-        key = this.sessions[i].participant_id
-        val = this.sessions[i].xnat_id
+      for (let i = 1; i < this.xnat2bids.sessions.length; i++) {
+        key = this.xnat2bids.sessions[i].participant_id
+        val = this.xnat2bids.sessions[i].xnat_id
         lines.push(`                       [${key}]="${val}"`)
       }
       const dictLine = lines.join(' \\\n')
       return dictLine + ')'
     },
 
     seriesDictString() {
-      let key = this.sessions[0].participant_id
+      let key = this.xnat2bids.sessions[0].participant_id
       if (!key) {
         key = 'MUST ENTER IN FORM'
       }
-      let sval = this.sessions[0].s_series.join(' -s ')
+      let sval = this.xnat2bids.sessions[0].s_series.join(' -s ')
       sval = sval.length > 0 ? '-s ' + sval : sval
 
-      let ival = this.sessions[0].i_series.join(' -i ')
+      let ival = this.xnat2bids.sessions[0].i_series.join(' -i ')
       ival = ival.length > 0 ? ' -i ' + ival : ival
 
       const lines = [`declare -A series_map=([${key}]="${sval + ival}"`]
-      for (let i = 1; i < this.sessions.length; i++) {
-        key = this.sessions[i].participant_id
-        sval = this.sessions[i].s_series.join(' -s ')
+      for (let i = 1; i < this.xnat2bids.sessions.length; i++) {
```

Comment:
this could probably be changed to `for (const session of this.xnat2bids.sessions)` and then could drop all the indexing (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of)